webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/DataTypes.scala

```diff
@@ -122,93 +126,88 @@ object PublicKey {
+final case class PublicKey(value: String) extends SecurityToken {
+  assert(!isEmpty(value))
 
   // Value of the key may be stored (with old fusion inventory version) as one line and without rsa header and footer, we should add them if missing and format the key
-  val key = {
+  val key =
     if (value.startsWith("-----BEGIN RSA PUBLIC KEY-----")) {
       value
     } else {
       s"""-----BEGIN RSA PUBLIC KEY-----
-      |${value.grouped(80).mkString("\n")}
-      |-----END RSA PUBLIC KEY-----""".stripMargin
+         |${value.grouped(80).mkString("\n")}
+         |-----END RSA PUBLIC KEY-----""".stripMargin
     }
-  }
-  def publicKey : IOResult[java.security.PublicKey] = {
+  def publicKey: IOResult[java.security.PublicKey] =
     IO.effect {
       (new PEMParser(new StringReader(key))).readObject()
     }.mapError { ex =>
       InventoryError.CryptoEx(s"Key '${key}' cannot be parsed as a public key", ex)
     }.flatMap { obj =>
       obj match {
-        case a : SubjectPublicKeyInfo =>
+        case a: SubjectPublicKeyInfo =>
           (new JcaPEMKeyConverter().getPublicKey(a)).succeed
         case _ => InventoryError.Crypto(s"Key '${key}' cannot be parsed as a public key").fail
       }
     }
-  }
 }
 
-final case class Certificate(value : String) extends SecurityToken { assert(!isEmpty(value))
+final case class Certificate(value: String) extends SecurityToken {
+  assert(!isEmpty(value))
 
   // Value of the key may be stored (with old fusion inventory version) as one line and without rsa header and footer, we should add them if missing and format the key
-  val key = {
+  val key =
     if (value.startsWith("-----BEGIN CERTIFICATE-----")) {
       value
     } else {
       s"""-----BEGIN CERTIFICATE-----
-      |${value.grouped(80).mkString("\n")}
-      |-----END CERTIFICATE-----""".stripMargin
+         |${value.grouped(80).mkString("\n")}
+         |-----END CERTIFICATE-----""".stripMargin
     }
-  }
-  def cert : IO[InventoryError, X509CertificateHolder] = {
+  def cert: IO[InventoryError, X509CertificateHolder] =
     for {
       reader <- IO.effect {
                   new PEMParser(new StringReader(key))
                 } mapError { e =>
                   InventoryError.CryptoEx(s"Key '${key}' cannot be parsed as a valid certificate", e)
                 }
-      obj    <- IO.effect(reader.readObject()).mapError { e =>
-                  InventoryError.CryptoEx(s"Key '${key}' cannot be parsed as a valid certificate", e)
-                }
-      res    <- obj match {
-                  case a : X509CertificateHolder =>
-                    a.succeed
-                  case _ => InventoryError.Crypto(s"Key '${key}' cannot be parsed as a valid certificate").fail
-                }
+      obj <- IO.effect(reader.readObject()).mapError { e =>
+               InventoryError.CryptoEx(s"Key '${key}' cannot be parsed as a valid certificate", e)
+             }
+      res <- obj match {
```

Comment:
it's a pity the <- are not aligned anymore

Reply:
it's repaired, ther are now.